src/ralph/accounts/admin.py

```diff
@@ -91,6 +92,21 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class StaffStatusFilter(BooleanFilter):
```

Comment:
in my opinion we should restore default django behaviour here for simple cases and use field name as @vi4m suggest. We'd not need metaclass for this - there is mechanism to register appropriate filter class for particular field type using `FieldListFilter` (see `django.contrib.admin.filters` for details). This would probably solve 90% of our cases and then we'd use custom filter classes when complex logic is required.